README1.md

```diff
@@ -26,7 +26,7 @@
 
 # 📖 'My Portfolio project <a name="about-project"></a>
 
-**My portfolio** is a project based on first mobile structure and it is first section of my portfolio
+**My portfolio** is a project based on first mobile structure and it is complete portfolio with even desktop responsive version.
```

Comment:
- [OPTIONAL] I would advise you to rename this file to readme.md not readme1.md, this is the reason your readme file is not showing in your repo.
- [OPTIONAL] I would implore you to add the key features section so that visitors can know the key features in your app. I strongly advise you do these things as they would affect your future milestone reviews.